addons/mail/static/src/models/partner/partner.js

```diff
@@ -190,6 +190,14 @@ function factory(dependencies) {
         // Private
         //----------------------------------------------------------------------
 
+        /**
+         * @private
+         * @returns {string}
+         */
+        _computeAvatarUrl() {
+            return `/web/image/res.partner/${this.id}/image_128`;
```

Comment:
For another PR but fallback for OdooBot could be here too `mail/static/src/img/odoobot.png`